sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1747,6 +1751,9 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(DiseaseConfiguration.class).executeRaw("UPDATE diseaseConfiguration SET caseFollowUpDuration = followUpDuration;");
 				getDao(DiseaseConfiguration.class).executeRaw("UPDATE diseaseConfiguration SET eventParticipantFollowUpDuration = followUpDuration;");
 
+			case 242:
+				currentVersion = 242;
+				getDao(Country.class).executeRaw("ALTER TABLE country ADD COLUMN archived SMALLINT DEFAULT 0;");
```

Comment:
@vlad-ciucescu I think the actual creation of the country table is missing when upgrading an existing mobile app. This needs to be added here. No idea why you have added the archived column to the table here. Maybe this is related to a state you had during development?

Reply:
I don't know what happened here either. I will add the table.

Reply:
Update: I did this like for the Region table
the table is created at line 253 `TableUtils.createTable(connectionSource, Country.class);`
and then I added the `archived` column, like it was added for Region at line 1219